pkg/report/table.go

```diff
@@ -192,6 +196,94 @@ func (tw TableWriter) setVulnerabilityRows(tableWriter *table.Table, vulns []typ
 		tableWriter.AddRow(row...)
 	}
 }
+func (tw TableWriter) setMisconfRows(table *tablewriter.Table, misconfs []types.DetectedMisconfiguration) map[string]int {
```

Comment:
Why was this function added?

Reply:
I pulled it accidentally while I resolved conflicts